bay_platform_dependencies.install

```diff
@@ -22,7 +22,7 @@ function bay_platform_dependencies_uninstall() {
 /**
  * Update section purger plugin to sectionbundled
  */
-function bay_platform_dependencies_update_8001() {
+function bay_platform_dependencies_update_10001() {
```

Comment:
This is fine as long as the release gets a major version bump and the notes mention the incompatibility with previous versions.

Reply:
yes this D10 will get a major version bump and also we have cleaned up all the old update hooks and the new ones starting from `10001` to represent the D10 version.